src/contract_wrappers/tokens/security_token_wrapper/__tests__/3.0.0.test.ts

```diff
@@ -0,0 +1,55 @@
+import { mock, instance, reset } from 'ts-mockito';
+import { Web3Wrapper, ISecurityTokenContract_3_0_0, FeatureRegistryEvents_3_0_0 } from '@polymathnetwork/abi-wrappers';
+import ContractFactory from '../../../../factories/contractFactory';
+import SecurityTokenCommon from '../common';
+import { SecurityToken_3_0_0 } from '../3.0.0';
+
+describe('SecurityToken 3.0.0', () => {
+  // we empty-extend the mixin to be able to use the class as a type
+  class FakeSecurityToken_3_0_0 extends SecurityToken_3_0_0 {}
```

Comment:
This is unnecessary in this case, and so is the preceding comment

You should be able to instance `SecurityToken_3_0_0` directly